fastlane/metadata/android/ca/title.txt

```diff
@@ -0,0 +1 @@
+Catima — Gestor de targetes de Fidelitat
```

Comment:
@mrestivill Please, please, please do not dismiss alerts.

I've written this down several times on the weblate page.

Google blocks translations that are too long